reduction/unite_contranges.py

```diff
@@ -119,14 +119,15 @@ def merge_contdotdat(field,band,basepath='/orange/adamginsburg/ALMA_IMF/2017.1.0
 
                 pairs = contrange.split(';') #split the contrange list into pairs
 
-                for p,pair in enumerate(pairs):#for each pair
+            for p,pair in enumerate(pairs):#for each pair
```

Comment:
just to verify quickly - is the change of indentation here intentional?

Reply:
No, emacs and copy-paste and github are not playing well together, even after I double-check the indentation by hand. I've fixed that now but will Slack you about a more reliable solution.